metadata-ingestion/src/datahub/ingestion/source/tableau.py

```diff
@@ -222,8 +227,22 @@ class TableauConfig(
     TableauConnectionConfig,
 ):
     projects: Optional[List[str]] = Field(
-        default=["default"], description="List of projects"
+        default=["default"],
+        description="[deprecated] Use project_pattern instead of projects. List of tableau "
+        "projects ",
+    )
+    # Tableau project pattern
+    project_pattern: AllowDenyPattern = Field(
+        default=AllowDenyPattern.allow_all(),
+        description="Regex patterns to filter tableau projects in ingestion. project_pattern may contain path to "
+        "nested project for ingestion, example A/B/C, here project C would get ingested",
     )
+
+    project_path_separator: str = Field(
+        default="/",
+        description="Projects path separator.",
```

Comment:
to make sure i understand: this is only relevant for how they format their project_pattern rules

Reply:
yes thats correct - beccause tableau projects can technically have any name